packages/ubuntu_desktop_installer/lib/pages/installation_type/installation_type_model.dart

```diff
@@ -74,22 +73,10 @@ class InstallationTypeModel extends SafeChangeNotifier {
   final productInfo = ProductInfoExtractor().getProductInfo();
 
   /// A list of existing OS installations or null if not detected.
-  List<OsProber>? get existingOS => _existingOS;
-
-  /// Initializes the model and queries the existing OS installation.
-  Future<void> init() async {
-    _diskService.getGuidedStorage().then((disks) {
-      _existingOS = disks
-          .expand<OsProber>(
-            (d) => d.partitions
-                .whereType<Partition>()
-                .map((p) => p.os)
-                .whereType<OsProber>(),
-          )
-          .toList();
-      notifyListeners();
-    });
-  }
+  List<OsProber>? get existingOS => _diskService.existingOS;
+
+  /// Initializes the model.
+  Future<void> init() async {}
```

Comment:
i'm leaving an empty method here because soon there will be initialization steps again :)